error_handling.py

```diff
@@ -0,0 +1,26 @@
+class CustomError(Exception):
+    def __init__(self, description: str, status_code: int, name: str):
+        self.description = description
+        self.status_code = status_code
+        self.name = name
+
+    def __str__(self):
+        return self.name
+
+
+class ValidationError(CustomError):
+    def __init__(self, description: str):
+        super().__init__(name="Bad request", status_code=400, description=description)
+
+    def __str__(self):
+        return self.name
+
+
+class WrongIdError(CustomError):
+    def __init__(self, description: str):
+        super().__init__(name="Not Found", status_code=404, description=description)
+
+    def __str__(self):
+        return self.name
```

Comment:
also, `__str__` is already implmented by base class, so extra `__str__` is not needed

Reply:
fixed